src/app-layer-htp.h

```diff
@@ -45,8 +45,7 @@
 #define HTP_CONFIG_DEFAULT_REQUEST_INSPECT_WINDOW       4096U
 #define HTP_CONFIG_DEFAULT_RESPONSE_INSPECT_MIN_SIZE    32768U
 #define HTP_CONFIG_DEFAULT_RESPONSE_INSPECT_WINDOW      4096U
-#define HTP_CONFIG_DEFAULT_FIELD_LIMIT_SOFT             9000U
```

Comment:
Was it unused ?

Reply:
Yes. libhtp never implemented soft limits so the soft limits were never actually used. This is mentioned in the commit message as well:

> htp_config_set_field_limits(soft_limit, hard_limit) changed to
> htp_config_set_field_limit(limit) because libhtp didn't implement soft
> limits.